pkg/endpoint/bpf.go

```diff
@@ -161,7 +161,7 @@ func (e *Endpoint) writeHeaderfile(prefix string, owner Owner) error {
 	fw.WriteString(common.FmtDefineAddress("LXC_MAC", e.LXCMAC))
 	fw.WriteString(common.FmtDefineAddress("LXC_IP", e.IPv6))
 	if e.IPv4 != nil {
-		fmt.Fprintf(fw, "#define LXC_IPV4 %#x\n", binary.BigEndian.Uint32(e.IPv4))
+		fmt.Fprintf(fw, "#define LXC_IPV4 %#x\n", byteorder.HostToNetworkSliceU32(e.IPv4))
```

Comment:
Well even before that PR it was doing  mix of htonl and not so it needs to be made consistent one way or the other. I don't mind if the BPF sidie does the conversion or the writeHeaderFile as long as it is done once.